### PR TITLE
allow a custom background to be used instead of the Minecraft parkour

### PR DESCRIPTION
**Pull request by reoccurcat**
Another edit lol, this time it's checking off one of the to-dos.
This commit is allowing a custom video background in place of the Minecraft parkour video. It also makes a couple of UI adjustments to keep up with the other commits done previously. The clips, instead of being named "background", are named off of their YouTube video ID (the parts following `?v=`, such as `n_Dv4JMiwK8`).

### CODE REVIEW

**Comment by jacesleeman** [comment]
instead of putting the url in background.py should be in .env or ask

**Comment by reoccurcat** [comment]
fixed conflicts

File: reddit/subreddit.py [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
 
     """
-    Returns a list of threads from the AskReddit subreddit.
+    Returns a list of threads from the provided subreddit.
     """
 
     load_dotenv()
 
-    print_step("Getting AskReddit threads...")
+    print_step("Getting subbreddit threads...")
```
Comment (CallumIO) [inline reddit/subreddit.py:16]: Spelling: Subreddit not Subbreddit

**Review by CallumIO — CHANGES_REQUESTED** [review]
Looks good, just need you to fix the spelling error.
Also could you remove the 'AskReddit' reference on line 34 of subreddit.py?

File: reddit/subreddit.py (hunk 1 above)
Comment (reoccurcat) [reply reddit/subreddit.py]: fixed!

**Comment by reoccurcat** [comment]
fixed the issues

**Comment by reoccurcat** [comment]
fixed conflicts again

**Comment by reoccurcat** [comment]
@CallumIO should be fixed now

**Comment by ProjectSlxsh** [comment]
Uploading WhatAreyouScaredof.mp4…
choped down so it fits githubs size requirements

**Comment by ProjectSlxsh** [comment]
https://user-images.githubusercontent.com/93690295/172453553-ab6b7d77-735f-4fda-85d3-f79ea8b1f359.mp4

cut down to 15-20 seconds to meet githubs size requirements

**Comment by reoccurcat** [comment]
It does that but saving the background using different file names each time so it doesn't get overwritten

**Comment by ProjectSlxsh** [comment]
ah I see, i dont understand what you did but, ok

**Comment by ProjectSlxsh** [comment]
also I know this is unrelated, but I did see that in the changes it mentioned changing the voice, how is this accomplished?

**Comment by reoccurcat** [comment]
at this point, I feel this pull request is being ignored. If no progress is made in another few days, I'll close it.

**Review by CallumIO — APPROVED** [review]
Sorry, it's not being ignored I've just been busy and we were waiting for a new release before any more big PR's - looks good though and thank you for this!

**Comment by reoccurcat** [comment]
Oh thank you!

**Pull request merged**